lib/push-manager.js

```diff
@@ -12,6 +12,8 @@ var providers = require('./providers');
 var loopback = require('loopback');
 var NodeCache = require('node-cache');
 var debug = require('debug')('loopback:component:push:push-manager');
+var SG = require('strong-globalize');
```

Comment:
Why not following strong-globalize coding guideline here?
https://www.npmjs.com/package/strong-globalize#autonomous-message-loading
Please study paragraphs after "For example, the following does not work as intended because the package sub calls SG.SetRootDir first:"